src/modules/taskEvents.js

```diff
@@ -0,0 +1,42 @@
+export default function createTaskDOM(desc) {
+  const newTask = document.createElement('li');
+  newTask.setAttribute('draggable', 'true');
+  newTask.className = 'task';
+  newTask.innerHTML = `<input type="checkbox" class="checkbox">
+    <input class="task-description" value="${desc}">
```

Comment:
- [ Required] Please consider adding an `event listener` that allows a user to edit the description of a `todo` item on the to-do list. When I tried to update a `todo` the change did not reflect in the local storage. You can achieve this by registering an `onchange` event listener on the `.task-description` input elements, then when a task description changes remember to update the local storage. Please see the video below for more details on this.

## Project requirement
![Screenshot from 2022-12-15 22-58-25](https://user-images.githubusercontent.com/16392046/207956828-dec78360-2ed2-4c46-885c-3c9fc86a04e2.png)

https://user-images.githubusercontent.com/16392046/207956513-a656de89-24fc-481a-ab3c-a85603407e99.mp4